python/MooseDocs/extensions/MooseBuildStatus.py

```diff
@@ -0,0 +1,87 @@
+import re
+import os
+from markdown.util import etree
+from markdown.util import AtomicString
```

Comment:
This is how you 'bypass' SmartyPants tostring routines. Allowing one to populate 
```text
<script> script content $%^&*"" </script>
```
With out losing logical operators and the like to HTML formatting.